Helper_Scripts/load_tests/chat_streaming_load.py

```
@@ -41,9 +41,56 @@
+        if "WORKFLOWS_EGRESS_ALLOWED_PORTS" not in os.environ:
+            port = parsed.port or (443 if parsed.scheme == "https" else 80)
+            os.environ["WORKFLOWS_EGRESS_ALLOWED_PORTS"] = f"{port},80,443"
+
+
+def _status_from_exc(exc: Exception) -> int:
+    resp = getattr(exc, "response", None)
+    if resp is not None:
+        try:
+            return int(getattr(resp, "status_code", 0) or 0)
+        except Exception:
+            pass
```

Ruff (0.14.10)

`try`-`except`-`pass` detected, consider logging the exception

(S110)